tests/content/input/17.handles deep nesting with mixed components.md

```diff
@@ -5,9 +5,10 @@
 ::::level-3
 ---
 foo: "bar"
+parent-foo:
```

Comment:
I added this parent prop since a key with a string as a value can't actually have a child property (it's not an object)